src/gui/main.cpp

```diff
@@ -6,6 +6,9 @@
 #include "neovimconnector.h"
 #include "app.h"
 
+#include <QtPlugin>
+Q_IMPORT_PLUGIN (QWindowsIntegrationPlugin);
+
```

Comment:
Wrap the include and Q_IMPORT in a guard for example

```C++
#if defined(Q_OS_WINDOWS) && defined(USE_STATIC_QT)
```

This would also require adding a definition in CMakeLists.txt

Reply:
Done